amiga-mui/configwnd.c

```diff
@@ -32,7 +32,7 @@
 #include <mui/BetterString_mcc.h>
 #include <mui/NListview_mcc.h>
 #include <mui/NListtree_mcc.h>
-#include <mui/popplaceholder_mcc.h>
+#include <mui/Popplaceholder_mcc.h>
```

Comment:
I had to revert this, as in http://aminet.net/dev/mui/MCC_Popph.lha all is lowercase. If there is a newer version available, in which the names of the includes are differently capitalized I'm happy to adjust the build scripts to download the newer version.

Reply:
yeah. i suspected it will remain to be a conflict. the usual problem is that such differences stay unnoticed and are not a subject of interest when software is considered to compile on an os that doesnt care about capitalization, like natively on amiga or its offshots. i have observed such include issues elsewhere, but it is easy to solve at compile time, so yours is certainly the best solution.